recipes/eigen/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release ..
+make check
```

Comment:
This gets a bit tricky. I don't remember how all the Travis CI VMs were affected by the transition, but some show way more cores than they actually have. So, we should be careful here.